src/Tribe/Linked_Posts.php

```diff
@@ -404,19 +403,27 @@ public function has_linked_posts_by_post_type( $post_id, $post_type ) {
 	public function get_linked_posts_by_post_type( $post_id, $post_type ) {
 		$result = array();
 
-		if ( $linked_post_ids = get_post_meta( $post_id, $this->get_meta_key( $post_type ) ) ) {
-			$args = array();
+		$args = array();
+
+		$linked_ids_order_key = $this->get_order_meta_key( $post_type );
+
+		if ( $linked_ids_order_key ) {
 			// Sort by drag-n-drop order
-			$linked_ids_order = get_post_meta( $post_id, $this->get_order_meta_key( $post_type ), true );
-			$linked_post_ids = tribe_sanitize_organizers( $linked_post_ids, $linked_ids_order );
+			$linked_ids_order = get_post_meta( $post_id, $linked_ids_order_key, true );
 			if ( ! empty( $linked_ids_order ) ) {
 				$args['post__in'] = $linked_post_ids;
 				$args['orderby'] = 'post__in';
 			}
+		}
 
-			$result = $this->get_linked_post_info( $post_type, $args, $linked_post_ids );
+		if ( ! empty( $linked_ids_order ) ) {
```

Comment:
I think on this one I would swap the code inside of the condition to avoid having the `!` (negation) operator there.

Into something like: 

```php
if ( empty( $linked_ids_order ) ) {
	$linked_post_ids = get_post_meta( $post_id, $this->get_meta_key( $post_type ) );
}
```

I think in this case you can also avoid the `else` block as you don't need to assign back to the same value.

Reply:
@mitogh if this is what you're suggesting:
```
		if ( empty( $linked_ids_order ) ) {
			$linked_post_ids = get_post_meta( $post_id, $this->get_meta_key( $post_type ) );
		}

		$result = $this->get_linked_post_info( $post_type, $args, $linked_post_ids );
```

Then `$linked_post_ids` wouldn't be set unless `$linked_ids_order` is empty.

`$linked_post_ids` is what gets sent to `$this->get_linked_post_info()`

Please let me know if I'm missing something. Thanks.

Reply:
Right, sorry for the confusion I updated my comments accordingly.

Reply:
cc @cliffordp